### DISCUSSION TITLE
pg_profile report is blank(Only title)

### DISCUSSION THREAD

**Discussion by beckom4** [zubkov-andrei/pg_profile#111] [Q&A]
Hi, 
We're working with PostgreSQL 14.12 on RHEL 9.2. 
We installed pg_stat_statements version 1.9 and dblink 1.2. 
Until now, we used pg_profile 4.1 without any issues. 
Today, we tried to update to pg_profile 4.7 and when it didn't work also to 4.6 and for both of them, the installation went ok, but when we tried to get a report it was blank and only showed the title. 
Do these versions support the dblink and pg_stat_statements versions mentioned above? 
Did anyone else encounter a similar issue? 
Thanks!

**Comment by zubkov-andrei** [comment]
Hi! Can you provide the report file, it looks like a bug, but it is difficult to locate it without report. If the report contains sensitive data, please provide the console output from browser developer tools.

**Comment by beckom4** [comment]
Hi!
My apologies for the delay, I somehow missed your latest response.
Unfortunately, the computers that we're working on are isolated from the web due to classification issues so there's no way for us to upload the files. I did take a picture with my phone though of how the issue looks like and of the console logs as requested, and if necessary, I'll be happy to take photos of the file's html/ properties or anything else you guys might require.
Another detail that might be important is that we tried opening it both with Microsoft Edge and with Google Chorme. Also, we checked the browsers' settings and they seem fine. The curious thing is that we tried to open the file on several PCs and in at least one of them we were able to open it.
![image](https://github.com/user-attachments/assets/1cbfe5da-1995-4f30-a244-318c103a8f7c)

![image](https://github.com/user-attachments/assets/4f8bf50a-b1ae-4a8e-a03d-c6100fc7c07f)

**Comment by beckom4** [comment]
Update:

We managed to fix the issue by upgrading both Microsoft Edge and Google Chrome to the newest versions. 

Thanks for your help!
